homeassistant/components/homekit/type_covers.py

```diff
@@ -334,6 +335,46 @@ def __init__(self, *args):
             *args, category=CATEGORY_WINDOW_COVERING, service=SERV_WINDOW_COVERING
         )
 
+@TYPES.register("Slat")
+class Slat(HomeAccessory):
+    """Generate a Slat accessory for a cover entity.
+
+    The entity must support: set_tilt_position.
+    """
+        
+    def __init__(self, *args):
+        super().__init__(*args, category=38) # https://docs.microsoft.com/en-us/dotnet/api/homekit.hmservicetype?view=xamarin-ios-sdk-12
```

Comment:
```suggestion
        super().__init__(*args, category=38, service=service) # https://docs.microsoft.com/en-us/dotnet/api/homekit.hmservicetype?view=xamarin-ios-sdk-12
```

It might not be working because the service isn't being passed up

Reply:
I started with that one indeed, but for easy testing I wrote this one from scratch eventually.
Thanks for having a look with me, appreciated! Perhaps there is another service that I could use where I can just pass the tilt orientation.

(or I will use the WindowCovering, where I map tilt_position to normal position).